package-lock.json

```diff
@@ -1,6483 +1,8 @@
 {
```

Comment:
Please remove this file from the PR 🙏🏼

Reply:
Removed . About combining grades , I think  that this could be useful and could be inserted as an extension of this button functionality in the future.

Reply:
To remove it from the PR, just push a version of your branch with the original `package-lock.json` file (the one we have in main).

Reply:
Do the same with the `dark-mode.css` please 🙏🏼

Reply:
Done.